internal/services/cognitive/cognitive_deployment_resource.go

```diff
@@ -149,7 +149,6 @@ func (r CognitiveDeploymentResource) Arguments() map[string]*pluginsdk.Schema {
 					"capacity": {
 						Type:         pluginsdk.TypeInt,
 						Optional:     true,
-						ForceNew:     true,
```

Comment:
Could we add a test with a non `OpenAI` type to validate this works for other types of accounts too?

Reply:
Sure, I will test that today

Reply:
@tombuildsstuff I am not able to perform this test because in `azurerm_cognitive_deployment`, the only supported format is `OpenAI` and it does not make sense to create a cognitive account of other type and a deployment with openai format. Link to the defination of format https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cognitive_deployment#format

Reply:
@liuwuliuyun we should still add an update test for this property with deployment type of `OpenAI` so that we can track if anything changes/regresses in the API.

Reply:
@stephybun Thanks for the review. Agreed on adding the testcase.